modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2PushedAuthRequestTestCase.java

```diff
@@ -211,4 +213,176 @@ private ApplicationResponseModel createApp() throws Exception {
 
         return getApplication(appId);
     }
+
+    private HttpResponse sendPostRequest(String endpoint, List<NameValuePair> parameters) throws Exception {
```

Comment:
can we use the existing `private String responsePost` method?

Reply:
Done with https://github.com/wso2/product-is/pull/21336/commits/52d60c6f61750b83024b86b4390f049149c7f366